controls/radrichtextbox/import-export/Html/settings.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Settings
+page_title: Settings
+description: Settings
+slug: radrichtextbox-import-export-html-settings
+tags: settings
+published: True
+position: 0
+---
+
+# Settings
+
+__HtmlFormatProvider__ allows for import of HTML documents and respectively export of RadRichTextBox to HTML. Additionally, the import/export settings provide modification options. The current article outlines the available settings.
+
+## Export Settings
+
+__HtmlFormatProvider__ exposes __ExportSettings__, which allow you to control the export of the RadRichTextBox document.
+
+### ExportSettings Properties
+
+* __DocumentExportLevel__: A property of type DocumentExportLevel that gets or sets which ELEMENT tags should be exported.
+	* __Document__: This is the default value. Includes the HTML declaration, more specifically &lt;HTML&gt;, &lt;TITLE&gt;, &lt;HEAD&gt; and &lt;BODY&gt; tags.
+	* __Fragment__: When set only the HTML &lt;BODY&gt; tag will be export.	
+* __ExportBoldAsStrong__: A property of type __bool__ that controls whether elements with font-weight bold are exported as <strong> tag.
+* __ExportEmptyDocumentAsEmptyString__: A property of type __bool__ that gets or sets if RadRichTextBox should export empty document as empty string.
```

Comment:
A property of type __bool__ that gets or sets whether an empty document should be exported as an empty string.